charts/apisix/values.yaml

```diff
@@ -68,6 +68,11 @@ apisix:
     # runAsNonRoot: true
     # runAsUser: 1000
 
+  # See https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more details
+  podDisruptionBudget:
+    enabled: false
+    maxUnavailable: 1
```

Comment:
I think the default value of `maxUnavailable` should also be given.

Reply:
should or should not? I'm not sure what's your meaning is.

Reply:
Oops, sorry, I mean the default value of "minAvailable".

Reply:
what's the default value you recommended?

Reply:
Maybe percents will be better, like `90%`?

Reply:
done